### PR TITLE
Bookstore Milestone 5

### PR DESCRIPTION
**Pull request by t-yanick**
In this PR we,
- Implemented styling of the various components based on the design provided at [this Zeplin Project](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)

Your kind review and feedback will help in improving the project.
**TAZOH Yanick TAZOH**
@t-yanick

### CODE REVIEW

File: src/components/App.css [hunk 1]
```diff
@@ -0,0 +1,252 @@
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+nav,
+ul {
+  display: flex;
```
Comment (Berabjesus) [inline src/components/App.css]: - [x] As an example here `display: flex;` is used on 11 different places. Kindly create a class for it and call it the react component when needed.

File: src/components/App.css [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+ul {
+  list-style-type: none;
+  width: 60%;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.panel-bg {
+  display: flex;
+  justify-content: space-around;
```
Comment (Berabjesus) [inline src/components/App.css]: - [x] `justify-content: space-around;` is used 6 different places. Kindly do the same for this and other duplicated rules.

**Review by Berabjesus — CHANGES_REQUESTED** [review]
# Status: CHANGES REQUIRED :warning: :construction: 
Hey @t-yanick! Congratulations on getting this far. You have done a remarkable job with this project. The styling of the web page is very well done 👏

### However, we need to take a step back and check out some CSS bad practices on this milestone.

- [x] In the stylesheets of the app, make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright). One requirement in this milestone is t avoid repeating the same rule. Please have a look at your CSS and refactor the duplicated rules. Have a look at the inline comments below.

#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**